<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>archived_orders_window</class>
 <widget class="QDialog" name="archived_orders_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1104</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1104</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order Tracker - Archived Orders</string>
  </property>
  <widget class="QPushButton" name="unarchive_order_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>113</width>
     <height>37</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Unarchive</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="archived_notes_edit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>401</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="archived_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>691</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Owner</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="current_orders_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>700</y>
     <width>161</width>
     <height>37</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Current Orders</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>archived_notes_edit</tabstop>
  <tabstop>archived_table</tabstop>
  <tabstop>current_orders_button</tabstop>
  <tabstop>unarchive_order_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
